python_ta/checkers/forbidden_import_checker.py

```diff
@@ -48,6 +48,15 @@ class ForbiddenImportChecker(BaseChecker):
                 "help": "Allow local modules to be imported.",
             },
         ),
+        (
+            "allowed-function-imports",
```

Comment:
Don't include a separate option; instead, both `allowed-import-modules` and `extra-imports` should allow both module names (e.g., `'math'`) and qualified functions (e.g., `math.floor`). Modify the `"help"` description of each of those options to refer to both modules and functions.